src/index.html

```diff
@@ -11,7 +11,213 @@
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="form">
+        <legend>Peronal infromation</legend>
+        <div class="form-field">
+          <label for="surname">
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              id="surname"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="name">
+            Name:
+            <input
+              type="text"
+              name="name"
+              id="name"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="age">
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              id="age"
+              value="12"
+              min="1"
+              max="100"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="dateOfBirth">
+            Full date of birth:
+            <input
+              type="date"
+              name="dateOfBirth"
+              id="dateOfBirth"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="acceptTerm">
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="acceptTerm"
+              id="acceptTerm"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form">
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label for="email">
+            E-mail:
+            <input
+              type="email"
+              placeholder="email@example.com"
+              name="email"
+              id="email"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="password">
+            Password:
+            <input
+              type="password"
+              name="password"
+              id="password"
+              required
+              minlength="8"
+              maxlength="50"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          <label>
+            Do you love cats?
+            
+            <label for="yes">
+              <input
+                type="radio"
+                name="cats"
+                value="yes"
+              >
+              Yes
+            </label>
+            <label for="no">
+              <input
+                type="radio"
+                name="cats"
+                value="no"
+              >
+              no
+            </label>
```

Comment:
and back here 3rd time
and the same question - does your label work? Nope.
@Edriss572 please, **test your app**

you're nesting an input inside label - it's already enough
- but you decide to use `for` attribute
   - okay, then what should it match against?
   - right, an `id` of the input, but you don't have one

```suggestion
            <label for="no">
            // either remove `for` OR provide corresponding `for` and `id`
            // it won't work otherwise
              <input
                type="radio"
                name="cats"
                value="no"
                id="no"
              >
              No
            </label>
```